src/main/capi/data_chunk-c.cpp

```diff
@@ -252,3 +252,16 @@ void duckdb_vector_reference_vector(duckdb_vector to_vector, duckdb_vector from_
 	auto dfrom_vector = reinterpret_cast<duckdb::Vector *>(from_vector);
 	dto_vector->Reference(*dfrom_vector);
 }
+
+const char *duckdb_data_chunk_to_string(duckdb_data_chunk chunk) {
+	auto dchunk = reinterpret_cast<duckdb::DataChunk *>(chunk);
+	auto str = dchunk->ToString();
+	auto result = static_cast<char *>(malloc(sizeof(char) * str.size()));
```

Comment:
This should probably be NULL terminated. Can we add a C API test where these methods are called to verify they work correctly?

Reply:
Good idea, done